core/server/api/v2/configuration.js

```diff
@@ -0,0 +1,67 @@
+const Promise = require('bluebird'),
```

Comment:
this should be expanded to `const` per each line ;) as in [other](https://github.com/TryGhost/Ghost/blob/master/core/server/api/v2/settings.js#L1-L10) controller